Mathlib/Data/Fin/Basic.lean

```diff
@@ -67,9 +67,6 @@ This file expands on the development in the core library.
   by `i ↦ n-(i+1)`
 -/
 
-assert_not_exists Monoid
-assert_not_exists Fintype
-universe u v
```

Comment:
is this by accident?

Reply:
Yes, it was certainly not intended: thanks for finding this!